### DISCUSSION TITLE
Option to delete chats

### DISCUSSION THREAD

**Discussion by LFd3v** [darrenburns/elia#64] [Ideas]
Hi, there.

Archive actually means the chat contents are deleted (removed from the database)? If not:

1. Is there a planned feature to browser and restore chats?
2. Maybe add a option to simply delete chats (with confirmation, if possible)?

Thanks again!. Regards

**Comment by rsipakov** [comment]
How can I restore a chat from an archive?

**Comment by feliperafaelbarbosa** [comment]
> How can I restore a chat from an archive?

I believe that just deleting it from sqlite.

**Comment by darrenburns** [comment]
> How can I restore a chat from an archive?

@rsipakov For now you'll have to set `archived=0` on the chat in the database. There's no way to do it via the UI.

> Is there a planned feature to browser and restore chats?

@LFd3v Yes, it's planned but not sure when I'll get to it. Could be a couple of months at least.

> Maybe add a option to simply delete chats (with confirmation, if possible)?

@LFd3v This would be part of the above feature. When viewing the archived chats you could a) restore or b) delete.
